docs/GettingStarted.md

```diff
@@ -1,19 +1,22 @@
 ## Introduction
-SnappyData fuses Apache Spark with an in-memory database to deliver a data engine capable of processing streams, transactions and interactive analytics in a single cluster.
+SnappyData fuses Apache Spark with an in-memory database to deliver a compute+data engine capable of stream processing, transactions, interactive analytics and prediction in a single cluster.
+
+!!! Attention:
+	This document assumes that you have familiarity with Apache Spark and its concepts. If you are new to Spark, refer to the [Spark documentation](https://spark.apache.org/docs/2.1.1/) to learn more about using Spark.
```

Comment:
Unless the version is highlighted prominently elsewhere, also add: The current release of SnappyData is fully compatible with Spark 2.1.1